mirrord-layer/tests/apps/fileops/src/main.rs

```diff
@@ -0,0 +1,54 @@
+#![feature(vec_into_raw_parts)]
+
+extern crate alloc;
+use alloc::ffi::CString;
+use std::{fs::OpenOptions, io::Write, os::unix::prelude::*};
+
+static FILE_CONTENTS: &str = "Hello, I am the file you're reading!";
+static FILE_PATH: &str = "/tmp/test_file.txt";
+
+fn create_test_file() {
+    println!(">> Creating /app");
```

Comment:
Ooops, forgot to change this to be like "Creating test file" (mb) (same in the similar test in e2e).